src/components/ClaimENSPage/ClaimENSPage.tsx

```diff
@@ -21,12 +67,17 @@ export default class ClaimENSPage extends React.PureComponent<Props, State> {
     this.setState({ name: data.value })
   }
 
-  handleOnAction = () => {}
+  handleOnAction = () => ({})
```

Comment:
You can send `undefined` to `onAction` and remove this :)
![image](https://user-images.githubusercontent.com/692077/100652773-2ce65b00-3326-11eb-8d5e-de9795e759e1.png)